test/testcafe/framework/page-objects/IdentityPageObject.js

```diff
@@ -199,12 +195,23 @@ export default class IdentityPageObject extends BasePageObject {
     await this.t.click(Selector(ENROLL_SELECTOR));
   }
 
+  getUnlockAccountLink() {
+    if(userVariables.v3) {
+      return this.form.getLink('Unlock account?');
+    }
+    return Selector(UNLOCK_ACCOUNT);
+  }
+
+  getUnlockAccountLinkText() {
+    return this.getUnlockAccountLink().textContent;
+  }
+
   async clickUnlockAccountLink() {
-    await this.t.click(Selector(UNLOCK_ACCOUNT));
+    await this.t.click(this.getUnlockAccountLink());
   }
 
   getCustomUnlockAccountLink() {
-    return Selector(UNLOCK_ACCOUNT).getAttribute('href');
+    return this.getUnlockAccountLink().getAttribute('href');
   }
```

Comment:
A tiny bit out of scope but wondering if it would cause a lot of file changes if you renamed this `getCustomUnlockAccountLinkUrl`?